src/web/src/components/Posts/Post.tsx

```diff
@@ -92,17 +82,80 @@ const useStyles = makeStyles((theme: Theme) =>
       lineHeight: '1.00',
       fontSize: '1em',
     },
+    titleAndAuthorContainer: {},
```

Comment:
Why bother with this if it's empty?

Reply:
Sorry, I miss this one during code clean-up.